apps/daimo-mobile/src/logic/nfcListen.ts

```diff
@@ -0,0 +1,69 @@
+import NfcManager, {NfcTech} from 'react-native-nfc-manager';
+import { MAX_NONCE_ID_SIZE_BITS } from "@daimo/userop";
+import { Hex } from "viem";
+import { generatePrivateKey } from "viem/accounts";
+import { useEffect } from "react";
+import { useNavigation } from "@react-navigation/native";
+import type { NativeStackNavigationProp } from "@react-navigation/native-stack";
+import { HomeStackParamList } from "../view/shared/nav";
+import { Recipient, getRecipient } from "../sync/recipients";
+
+NfcManager.start();
+
+
+async function readNdef(nav: NativeStackNavigationProp<HomeStackParamList>) {
```

Comment:
i'd add a short docstring on this function and on the exported `useInitNfc` that summarize what they do